pkg/cri/server/container_status.go

```diff
@@ -103,6 +107,11 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 		st, ft = status.StartedAt, status.FinishedAt
 	}
 
+	runtimeSpec, err := container.Container.Spec(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get container runtime spec: %w", err)
```

Comment:
let's create the runtime.ContainerStatus first then get the spec, log any error to debug, otherwise create the Resources field using the spec, then return result...

Reply:
if we don't want to return errors, should we at least log errors to error instead of debug, just for better visibility?

Reply:
log error is fine.. is already being logged as an error below in verbose info so might get logged twice if it ever happens depending on verbosity